pkgs/applications/misc/input-leap/default.nix

```diff
@@ -1,46 +1,57 @@
-{ lib, fetchFromGitHub, cmake, curl, xorg, avahi, qt5, openssl, stdenv
+{ lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, curl
+, xorg
+, avahi
+, qtbase
+, qttools
+, wrapQtAppsHook
+, openssl
+, stdenv
 , wrapGAppsHook
 , avahiWithLibdnssdCompat ? avahi.override { withLibdnssdCompat = true; }
-, fetchpatch }:
+, fetchpatch
+}:
 
 stdenv.mkDerivation rec {
   pname = "input-leap";
-  version = "2.4.0";
+  version = "unstable-2023-02-25";
 
   src = fetchFromGitHub {
-    owner = "input-leap";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-2tHqLF3zS3C4UnOVIZfpcuzaemC9++nC7lXgFnFSfKU=";
+    owner = "KoviRobi";
+    repo = "input-leap";
+    rev = "62ff806e08bdd2a06973175cbd36f4b08b9803ef";
+    sha256 = "lrNnvIbupK19ECy47Sffjb8zR4E35GxcQTdoae1Qnd0=";
```

Comment:
I've fixed (I think, I use `i3` so I don't use desktop icons) the `.desktop` file, see https://github.com/input-leap/input-leap/pull/1609